packages/x-charts/src/SparkLineChart/SparkLineChart.tsx

```diff
@@ -181,23 +192,35 @@ const SparkLineChart = React.forwardRef(function SparkLineChart(
     className,
     disableClipping,
     clipAreaOffset,
+    onHighlightChange,
+    onHighlightedAxisChange,
+    highlightedAxis,
+    highlightedItem,
     ...other
   } = props;
   const id = useId();
   const clipPathId = `${id}-clip-path`;
-  const clipPathOffset = {
-    top: clipAreaOffset?.top ?? 1,
-    right: clipAreaOffset?.right ?? 1,
-    bottom: clipAreaOffset?.bottom ?? 1,
-    left: clipAreaOffset?.left ?? 1,
-  };
+  const clipPathOffset = React.useMemo(
```

Comment:
While I don't have an issue with having them here, shouldn't these memoization be moved to the plugins or series config? 🤔

Reply:
> Those default are chart specific

Ah true, because the sparkline uses the line/bar series config 😢
I've been a bit too focused on the sankey lately which is the chart/series 😆